### PR TITLE
simplePinkTheme

### PR DESCRIPTION
**Pull request by merceavanessa**

### CODE REVIEW

File: examples/ligo.html [hunk 1]
```diff
@@ -4,17 +4,16 @@
 <head>
     <title>32 seconds around the GW150914 event</title>
 
-    <link rel="stylesheet" href="../node_modules/ni-webcharts/styles/webchartsLight.css" />
     <link rel="stylesheet" href="../node_modules/jqwidgets-framework/jqwidgets/styles/jqx.base.css" />
     <link rel="stylesheet" href="../node_modules/jqwidgets-framework/jqwidgets/styles/jqx.fresh.css" />
     <link rel="stylesheet" href="../node_modules/ni-webcharts-legends/styles/webcharts-legends.icons.css" />
     <link rel="stylesheet" href="../node_modules/ni-webcharts-legends/styles/webcharts-legends.css" />
+    <link rel="stylesheet" href="../something.css" />
```
Comment (ralucaportase) [inline examples/ligo.html]: You should rename the css theme to something more sugestive

File: node_modules/ni-webcharts/styles/webchartsDark.css [hunk 2]
```diff
@@ -7,11 +7,11 @@
 ni-cartesian-graph,
 ni-chart,
 ni-intensity-graph {
-    border: 1px solid #575C60;
+    border: 1px solid #D2B4B4;
```
Comment (cipix2000) [inline node_modules/ni-webcharts/styles/webchartsDark.css]: revert these changes

File: examples/ligo.html (hunk 1 above)
Comment (cipix2000) [inline examples/ligo.html]: like simplePink.css, or darkPink.css or maybe something more artistically like smokedPink or somethin'

Comment (cipix2000) [inline examples/ligo.html]: or fiftyShadesOfPink

File: something.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
+body {
+  background-color: #303030;
+}
+
+.hyperL {
```
Comment (cristiingineru) [inline something.css]: hyperLoop? :1st_place_medal: 
Let's try something more precise like hyperlink?

File: something.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  color: #CA4BCB;
+}
+
+.hCustom {
```
Comment (cristiingineru) [inline something.css]: No need to define a new header class. Apply these styles to the h1 tag name.

File: fiftyShadesOfPink.css [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+ni-cartesian-graph,
+ni-chart,
+ni-intensity-graph {
+      border: 1px solid #575C60;
+      padding: 20px;
+      display: inline-block;
+      min-height: 20px;
+      color: #d2b4b4;
+      background-image: url("/examples/peace.png");
```
Comment (cristiingineru) [inline fiftyShadesOfPink.css]: Use relative path:
```
url("./examples/peace.png");
```

You can see the error (among many other non related errors) here:
https://rawgit.com/ni-kismet/webcharts-drive-test/test/examples/ligo.html

Note where the test page is trying to load your peace from:
```
GET https://rawgit.com/examples/peace.png 404 ()
```
instead of:
```
https://rawgit.com/ni-kismet/webcharts-drive-test/test/examples/peace.png
```

Comment (merceavanessa) [inline fiftyShadesOfPink.css]: Am crezut că dacă pun asa nu mai e relativ path-ul. O sa caut.

**Comment by cristiingineru** [comment]
There is one more change that has to be reverted:

node_modules/ni-webcharts/styles/webchartsDark.css

File: fiftyShadesOfPink.css [hunk 6]
```diff
@@ -40,6 +40,9 @@ ni-intensity-graph {
       min-height: 20px;
       color: #d2b4b4;
       background-image: url("./examples/peace.png");
+      background-repeat: no-repeat;
+      background-position: 100% 100%;
```
Comment (cristiingineru) [inline fiftyShadesOfPink.css]: :+1:

**Comment by cristiingineru** [comment]
:1st_place_medal:
